backend/app.py

```diff
@@ -740,6 +786,23 @@ def deleteReport(report_id):
         return jsonify({"error": "An unexpected error occurred. Please try again later."}), 500
 
 
+
+# Get report type "curation" and "companySummarization"
+@app.route("/api/reports/curation/all", methods=["GET"])
+def getAllReportsCuration():
+    """
+    Endpoint to get all reports type "curation" or "companySummarization".
+    """
+    try:
+        reports = get_all_reports_curation()
```

Comment:
@ramirezmorac2 This function should support filtering by type. If we go with option 2, it will simply pass "curation" or "summarization" as a parameter.